src/dos/dos_locale.cpp

```diff
@@ -1442,6 +1926,26 @@ static const std::map<Country, CountryInfo> CountryData = {
 			Separator::Semicolon      // list separator
 		} }
 	} } },
+	{ Country::SaudiArabia, { "Saudi Arabia", "SAU", {
+		{ LocalePeriod::Modern, {
+			// (taken from the common Arabic, adapted the currency)
+			DosDateFormat::DayMonthYear, Separator::Period,
```

Comment:
Should be ar_SA
https://sourceware.org/git/?p=glibc.git;a=blob;f=localedata/locales/ar_SA;h=7b92faa3d38b2106a2155fa774ddc19b4c1ebd36;hb=HEAD